setup.cfg

```diff
@@ -45,10 +45,12 @@ project_urls =
 [options]
 python_requires = >=3.8
 zip_safe = False
-# install_requires =
-#     numpy>=1.19.0
-#     numexpr>=2.6.2
-#     packaging
+install_requires =
+    numpy>=1.19.0
+    numexpr>=2.6.2
+    blosc2~=0.6.6
```

Comment:
Strictly speaking blosc2 (intended as pytthon-blosc2) is not a requirement (cblosc2 is).
In principle cblosc2 could be provided as system library.

This is not a big issue IMHO bur maybe it could make sense to add a small note in the documentation or at least a comment to explain the situation.

Reply:
Ok, I can add a note to the dev guide and to file!

At the moment, even before this PR, it already seems like our wheels have a hard dependency on everything in the https://github.com/PyTables/PyTables/blob/master/requirements.txt file because it's all listed in the record as a dependency. I'm not sure where that is coming from. Does pip automatically look at the requirements file as a list of dependencies?

Reply:
Some context here. By adopting dependency on Python-Blosc2 wheels I wanted to avoid the different issues in supporting plain C-Blosc2 builds. Python-Blosc2 comes with high quality C-Blosc2 binary libs, and we (the Blosc team) plan doing so forever. So, for the sake of simplicity, I'd like to continue using C-Blosc2 *only* via Python-Blosc2.

Regarding the dependency of requirements.txt, I think it was mainly my fault.  I just wanted to be more consistent in all the building yaml files, and decided to go this route. If that is creating issues, we can go back here.

Reply:
As I said having python-blosc2 listed as a requirement is not a big issue for me.
After all it is a small package and the current solution simplifies a lot of stuffs.

I suggested to document it somewhere mostly to help a little bit people that creates packages that are not based on wheels.